R/google_drive.R

```diff
@@ -166,30 +171,50 @@ gd_put <- function(
   }
   
   # post the file (create or update) from the local data file to Google Drive
+  remote_file_changed <- TRUE # assume we'll be changing it
   if(is.na(remote_id)) {
     if(verbose) message("Uploading ", local_file, " to Google Drive")
     remote_id <- googledrive::drive_upload(media=local_file, path=googledrive::as_id(parent), type=type, verbose=verbose)$id
   } else {
     on_exists <- match.arg(on_exists)
-    switch(
-      on_exists,
-      update={
-        if(verbose) message("Updating ", local_file, " on Google Drive")
-        remote_id <- googledrive::drive_update(googledrive::as_id(remote_id), media=local_file, verbose=verbose)$id
-      },
-      replace={
-        if(verbose) message("Replacing ", local_file, " on Google Drive")
-        googledrive::drive_rm(googledrive::as_id(remote_id), verbose=verbose)
-        remote_id <- googledrive::drive_upload(media=local_file, path=googledrive::as_id(parent), type=type, verbose=verbose)$id
-      },
-      stop={
-        stop('File already exists and on_exists==stop')
+    if(on_exists == 'stop') {
+      stop('File already exists and on_exists==stop')
+      # remote_file_changed <- FALSE # true but unnecessary
```

Comment:
did you intend to leave this?

Reply:
well...yes, just for clarity about the meaning of `remote_file_changed`, but you're right to imply that it's sorta silly to have it in there. i'll take it out.